src/Workspaces/Core/Portable/CodeActions/Annotations/SuppressDiagnosticsAnnotation.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.CodeActions
+{
+    internal class SuppressDiagnosticsAnnotation
+    {
+        public const string Kind = "CodeAction_SuppressDiagnostics";
+
+        public static SyntaxAnnotation Create()
+        {
+            return new SyntaxAnnotation(Kind);
```

Comment:
It likely could be.  However, i was thinking that in the future we might add support for people to suppress specific diagnostics.  So given that we could move to non-singletons in the future, i thought it might be premature to start with a singleton.

I could keep 'Create' but have it return a singleton.  But that seemed a little wonky to me.  But if you'd prefer that, i have no issue.  Note: as this is literally a single object created in a single user-invoked, feature-codepath, i didn't think the savings warrented anything :)